README.md

```diff
@@ -2,130 +2,134 @@
 
 # Kata Containers
 
-* [Raising issues](#raising-issues)
-* [Kata Containers repositories](#kata-containers-repositories)
-    * [Code Repositories](#code-repositories)
-        * [Kata Containers-developed components](#kata-containers-developed-components)
-            * [Agent](#agent)
-            * [KSM throttler](#ksm-throttler)
-            * [Runtime](#runtime)
-            * [Trace forwarder](#trace-forwarder)
-        * [Additional](#additional)
-            * [Kernel](#kernel)
-    * [CI](#ci)
-    * [Community](#community)
+* [Kata Containers](#kata-containers)
+    * [Introduction](#introduction)
+    * [Getting started](#getting-started)
     * [Documentation](#documentation)
-    * [Packaging](#packaging)
-    * [Test code](#test-code)
-    * [Utilities](#utilities)
-        * [OS builder](#os-builder)
-    * [Web content](#web-content)
+    * [Community](#community)
+    * [Getting help](#getting-help)
+        * [Raising issues](#raising-issues)
+            * [Current version](#current-version)
+            * [Kata Containers 1.x legacy versions](#kata-containers-1x-legacy-versions)
+    * [Developers](#developers)
+        * [Components](#components)
+            * [Legacy components](#legacy-components)
+        * [Common repositories](#common-repositories)
+    * [Credits](#credits)
 
 ---
 
 Welcome to Kata Containers!
 
-The purpose of this repository is to act as a "top level" site for the project. Specifically it is used:
-
-- To provide a list of the various *other* [Kata Containers repositories](#kata-containers-repositories),
-  along with a brief explanation of their purpose.
-
-- To provide a general area for [Raising Issues](#raising-issues).
+This repository is the home of the current Kata Containers code.
```

Comment:
I would suggest calling out 2.0 explicitly here to be clear- maybe "home of the Kata Containers 2.0 code"

Reply:
It would be a bit awkward as we'll end up having to update the sentence again when we release `2.1` etc.

Reply:
Yes, it's awkward to get the wording clear here. We could say something like the following, but it feels a little "unwieldy" to me (might confuse new users who "just want the latest" and don't care about the history :smile::

> This repository is the home of the Kata Containers version 2.0 and higher code.